electroncash_gui/qt/request_list.py

```diff
@@ -155,5 +157,5 @@ def create_menu(self, position):
         menu.addAction(_("Copy URI"), lambda: self.parent.view_and_paste('URI', '', self.parent.get_request_URI(addr)))
         menu.addAction(_("Save as BIP70 file"), lambda: self.parent.export_payment_request(addr))
         menu.addAction(_("Delete"), lambda: self.parent.delete_payment_request(addr))
-        run_hook('receive_list_menu', menu, addr)
+        run_hook('receive_send_via_email_list_menu', menu, addr, self.parent.get_request_URI(addr), self.parent.receive_qr, status) 
```

Comment:
Hmm. This hook seems oddly-specific. The hooks are intended to be generic mechanisms. 

I hesitate to change the old "hooks" API for this. Especially since the old hook was very generic and maybe some plugin out there would want to use it. I suggest we actually just update the API to more closely match what electrum is doing now -- they modified this hook to work as follows.

Also you don't really need to pass all that stuff in -- the callee can determine it anyway.

```suggestion
        run_hook('receive_list_menu', self.parent, menu, addr)
```